tests.py

```diff
@@ -208,3 +208,81 @@ def test_sip_api_already_set():
             import sip
             sip.setapi("QString", 1)
             assert_raises(ImportError, __import__, "Qt")
+
+
+def test_translate_and_UnicodeUTF8_in_pyside():
+    with pyside():
+        from Qt import QtWidgets
+
+        # this does exist in PySide
+        assert QtWidgets.QApplication.UnicodeUTF8 is not None
+
+        # use patched method with old arguments
+        QtWidgets.QApplication.translate(
```

Comment:
cool I'll have a look, if you don't mind though , for now, I'll hammer a bit your remote testing ;)